chapters/Chapter 1 — Untitled.md

```diff
@@ -49,14 +49,76 @@ Something wet was pooling at the bottom of her helmet’s goggles. Right: she wa
 
 With the gravity field she should be good to run, but she kept her pace steady as she moved through her bedroom to the door of the hold, carefully ducking under the flying crate and batteries.
 
-Her hand reached for her gun as she opened the door, but there was no need. Her attacker was tumbling. His neck wasn’t broken yet: he was still managing to bring his arms up in time to protect himself, but he was tiring embarrassingly quickly. A few of his many weapons had detached themselves from him and were now flying around the hold with him.
+Her hand reached for her gun as she opened the door, but there was no need. Her attacker was tumbling. His neck wasn’t broken yet: he was still managing to bring his arms long up in time to protect himself, but he was tiring embarrassingly quickly. A few of his many weapons had detached themselves from him and were now flying around the hold with him.
 
 The hold door resealed itself behind Leanne. She must’ve flicked the switch out of habit—it was a good habit when on a spaceship.
 
 She’d closed it just in time. The forces of the spinning ships were far too much for their docking connectors to handle, and they wrenched themselves apart.
 
 Her attacker had left the airlocks open in case he had needed a quick escape. His escape was rather quicker than he intended: the air from the hold forced its way out, shoving him into space.
 
-Re: assassin flying out of airlock, and her after him:
+Leanne was not far behind: her gravity generator tried to keep her on the floor; her fingers brushed the griprail; it was not enough. 
 
-Later, she’d tell herself she intended it this way all along. But she more preferred the simpler, direct approach. Long-term gambles rarely paid.
+Later, she’d tell herself she intended it this way all along. She’d put the suit on, after all.
+
+Had she not been wearing the generator, Leanne assumed she’d be spinning fast. Instead, she was floating calmly through the apparent emptiness, breathing fresh oxygen from the tanks on her back.
+
+Her ship was in a desperate spin. The engines had stopped: the ship was just smart enough to detect it was spinning, but not quite smart enough to stop the spin. She’d figure out how to stop it later.
+
+The other ship was twice as large as hers. It looked patched together from pieces of other ships, but Leanne was rather certain it had been designed that way: it had been the style in the 2870’s, and was still popular amongst some. Its seven side engines seemed rather weak; Leanne supposed most of the power was reserved for the rear engines. She looked away as they flashed bright green. 
```

Comment:
"... and was still popular amongst some" could sound more dismissive. Freaking hipsters.

"... and was still all the rage amongst some" "... and it was still popular with some, though Leanne could hardly understand why."

"... and Leanne guessed some still found it popular."

That last one is not bad; "Leanne guessed," "Leanne assumed," etc. is a thing in this story. She assumes a lot, because she never feels like she knows anything. She feels like she's awash in uncertainty at all times.